src/components/VButton/VButton.vue

```diff
@@ -0,0 +1,17 @@
+<template>
+  <button class="button" v-on="$listeners" :bisable="disable"  >
```

Comment:
Should probably be `disabled`?  
But consider not explicitly defining this prop as just passing `disabled` to this component would apply it to it automatically.